reconcile/gitlab_housekeeping.py

```diff
@@ -165,16 +166,17 @@ def merge_merge_requests(dry_run, gl, merge_limit, rebase, insist=False):
             if not pipelines:
                 continue
 
-            # possible statuses:
-            # running, pending, success, failed, canceled, skipped
-            incomplete_pipelines = \
-                [p for p in pipelines
-                 if p['status'] in ['running', 'pending']]
-            if incomplete_pipelines:
-                if insist:
-                    raise Exception(f'insisting on {merge_label}')
-                else:
-                    continue
+            if wait_for_pipeline:
```

Comment:
Shouldn't we wait for running?

Reply:
as long as the last one passed, we don't really care about previous ones...
plus, the disconnection may also mean that jobs that are "running" are not really running anymore.

Reply:
got it